lighthouse-core/lib/i18n/locales/en-XL.json

```diff
@@ -1253,9 +1259,15 @@
   "lighthouse-core/lib/lh-error.js | dnsFailure": {
     "message": "D̂ŃŜ śêŕv̂ér̂ś ĉóûĺd̂ ńôt́ r̂éŝól̂v́ê t́ĥé p̂ŕôv́îd́êd́ d̂óm̂áîń."
   },
+  "lighthouse-core/lib/lh-error.js | erroredRequiredArtifact": {
+    "message": "R̂éq̂úîŕêd́ {artifactName} ĝát̂h́êŕêŕ êńĉóûńt̂ér̂éd̂ án̂ ér̂ŕôŕ: {errorMessage}"
+  },
```

Comment:
Not sure where this (and the blob below) came from.

Reply:
> Not sure where this (and the blob below) came from.

master was out of date. #9404 fixed this so it should disappear if you rebase or merge